content/merchants/index.md

```diff
@@ -19,7 +19,17 @@ type: "default"
 <p>Anyone can accept NavCoin on their website or in their store to receive instant global payments.</p>
 <p>If your store accepts NavCoin and you would like to be listed on this page, read about how you can contribute to this website.
 {{< /left_image_section >}}
-
+    
+{{< home_centre_text_section
+    title="World map of all stores and merchants accepting NAV"
+    linkBgColor="#43799a"
+    gradient=""
+    fontColor="#FFF">}}
+    <div>
+      <iframe src="https://www.google.com/maps/d/embed?mid=1wzMJAyujqSQ-Iyy0mqQxP_uuwrphIuYJ&hl=en" width="960" height="720"></iframe>
+    </div>
+    {{< /home_centre_text_section>}}
+    
 {{< links_section
```

Comment:
We should also add some instruction of how to add your store to the map via the google form (or talking to mods on discord) and link to the form.